src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -31,9 +31,10 @@ abstract class FileLoader extends Loader
 
     private $currentDir;
 
-    public function __construct(FileLocatorInterface $locator)
+    public function __construct(FileLocatorInterface $locator, string $env = null)
     {
         $this->locator = $locator;
+        $this->env = $env;
```

Comment:
Could we call `parent::__construct($env)`? This made my head spin when I first saw it - it's just not the most natural pattern (even though it's legal).

Reply:
thanks for the review
updted :)